uiowa/uiowa
# Customizable research areas label

Resolves #5876

Label adjustment appears on:

- [x] Rendered person node
- [x] /people "Research Areas" exposed filter label
- [x] Update /admin/config/sitenow/sitenow-people 



### Next steps:
- [ ] Deploy with current PR
- [ ] Update sites manually to "Service Areas" for Law Library and "Practice Areas" for Pharmacy
- [ ] Remove patched config for Pharmacy

### How to Test:
1. `blt ds --site=default`, `blt ds --site=lawlibrary.uiowa.edu`, `blt ds --site=pharmacy.uiowa.edu`
2. Head to /admin/config/sitenow/sitenow-people on all 3 sites.
3. The "Research Areas" label should read "Override for Research Areas label" as should the filters (see screenshot), the value should default to "Research Areas". The filters should say "Research Areas".
![image](https://github.com/uiowa/uiowa/assets/52135878/b232bc9c-5725-42f9-9635-babb7091e82b)

4. Update all sites' labels to something else (or text from above) and see if the people node label updates too.
  https://lawlibrary.uiowa.ddev.site/people/brock-neff
  https://default.uiowa.ddev.site/people/herky-hawk
  https://pharmacy.uiowa.ddev.site/sitenowpeople/marianne-carter
